docs/.vitepress/theme/components/sandbox/SandboxPreview.vue

```
@@ -18,8 +18,8 @@ onMounted(() => {
 
 // Scroll into the operation when the operationId changes.
 watch(sandboxData.operationId, () => {
-  if (sandboxData.previewComponent.value === 'OAOperation') {
-  } else if (sandboxData.previewComponent.value === 'OASpec') {
+  if (sandboxData.previewComponent.value === 'PagesByOperation') {
+  } else if (sandboxData.previewComponent.value === 'PagesBySpec') {
```

ESLint

[error] Empty block statement.

(no-empty)